e2e/cypress/integration/enterprise/cloud/billing/subscriptions_free_trial_spec.js

```diff
@@ -0,0 +1,260 @@
+                    cy.request(link.prop('href')).its('status').should('eq', 200);
+                });
+            });
+    });
+
+    it('MM-37054 - "Contact Sales" navigation from Subscribe window', () => {
+        // # Click on Subscribe Now button
+        cy.contains('span', 'Subscribe Now').parent().click();
+
+        // * Check for "Provide Your Payment Details" label
+        cy.get('.title').contains('span', 'Provide Your Payment Details').should('be.visible');
+
+        // * Check for Contact Sales navigation
+        cy.window().
+            its('store').
+            invoke('getState').
+            then((state) => {
+                const currentiuserid = state.entities.users.currentUserId;
+                const email = encodeURIComponent(state.entities.users.profiles[currentiuserid].email);
+                const name = encodeURIComponent(`${state.entities.users.profiles[currentiuserid].first_name} ${state.entities.users.profiles[currentiuserid].last_name}`);
+                cy.contains('span', 'Contact Sales').parent().then((link) => {
+                    const getHref = () => link.prop('href');
+                    cy.wrap({href: getHref}).invoke('href').should('contains', `/contact-us?email=${email}&name=${name}&inquiry=sales`);
+                    cy.wrap(link).should('have.attr', 'target', '_blank');
+                    cy.wrap(link).should('have.attr', 'rel', 'noopener noreferrer');
+                    cy.request(link.prop('href')).its('status').should('eq', 200);
+                });
+            });
+    });
+
+    it('MM-T4128 - Enabling of "Subscribe" button in Subscribe window', () => {
+        // # Click on Subscribe Now button
+        cy.contains('span', 'Subscribe Now').parent().click();
+
+        // * Check for "Provide Your Payment Details" label
+        cy.get('.title').contains('span', 'Provide Your Payment Details').should('be.visible');
+
+        // # Enter card details
+        getIframeBody().find('[name="cardnumber"]').clear().type('4242424242424242');
+        getIframeBody().find('[name="exp-date"]').clear().type('4242');
+        getIframeBody().find('[name="cvc"]').clear().type('412');
+        cy.get('#input_name').clear().type('test name');
+        cy.contains('legend', 'Country').parent().find('.icon-chevron-down').click();
+        cy.contains('legend', 'Country').parent().find("input[type='text']").type('India{enter}');
+        cy.get('#input_address').type('test1');
+        cy.get('#input_address2').type('test2');
+        cy.get('#input_city').clear().type('testcity');
+        cy.get('#input_state').type('test');
+        cy.get('#input_postalCode').type('444');
+
+        // * Check for enable status of Subscribe button
+        cy.get('.RHS').find('button').should('be.enabled');
+    });
+
+    it('MM-T4129 - Disabling of Subscribe button for not having valid data in Manditory fields in Subscribe screen', () => {
```

Comment:
Typo error check. Also, please remove ` - ` between test ID and title.

```suggestion
    it('MM-T4129 Disabling of Subscribe button for not having valid data in Mandatory fields in Subscribe screen', () => {
```

Reply:
corrected typo and removed "-" in all the test cases